src/api/middlewares/errorHandler.ts

```diff
@@ -0,0 +1,14 @@
+import type { Request, Response, NextFunction } from "express";
+import type { CustomError } from "../types/error.js";
+
+export default function errorHandler(err: CustomError, req: Request, res: Response, next: NextFunction) {
+  console.error(err.stack);
+
+  if (res.headersSent) return next(err);
+
+  if (err.status) {
+    res.status(err.status).json({ error: err.message });
```

Comment:
서비스에서는 `error.statusCode`를 리턴하는데 비해 `err.status`를 참조하면 올바르지 않은 것 같습니다. 올바른 변수를 참조하도록 해주세요
```suggestion
  if (err.statusCode ?? err.status) {
    res.status(err.status).json({ error: err.message });
```